contrib/gocql/gocql/example_test.go

```diff
@@ -7,15 +7,18 @@ package gocql_test
 
 import (
 	"context"
+	"log"
+
+	"github.com/gocql/gocql"
 
 	gocqltrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gocql/gocql"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
-func Example() {
+func ExampleNewCluster() {
 	// Initialise a wrapped Cassandra session and create a query.
-	cluster := gocqltrace.NewCluster([]string{"127.0.0.1"}, gocqltrace.WithServiceName("ServiceName"))
+	cluster := gocqltrace.NewCluster([]string{"127.0.0.1:9043"}, gocqltrace.WithServiceName("ServiceName"))
```

Comment:
Not directly related to the content of *this* specific PR but I know internally, at Datadog, we do wrap this and try to *NOT* use `WithServiceName` but instead rely on `ext.PeerService`. This PR https://github.com/DataDog/dd-trace-go/pull/1975 has more context. I am not sure I fully understand why `ext.PeerService` is superior to a plain Service, but it looks like it is.

Mentioning this as since we're introducing a (non-breaking, which is great) API change, it may be the right time.